src/consent.js

```diff
@@ -79,6 +79,28 @@ export function getConsentPolicyInfo(element, policyId) {
   );
 }
 
+/**
+ * TODO(micajuine-ho): Combine with getConsentPolicyGdprApplies
+ * (and getConsentType in future) to return a consentMetadata
+ * object.
+ * @param {!Element|!ShadowRoot} element
+ * @param {string} policyId
+ * @return {!Promise<?Object|undefined>}
+ */
+export function getConsentMetadata(element, policyId) {
```

Comment:
Given the metadata always comes with the consent string. What do you think about returning something like? We don't need to finalize the API in this PR, but providing a thought
```
{
  string,
  metadata
}
```

Reply:
Are there some situations where we only need the consentString and not the additional metadata? 
One use case that comes to mind is the current CONSENT_STRING macro. That would be a reason to keep them separate.